src/main/frontend/components/whiteboard.css

```
@@ -212,3 +212,13 @@ input.tl-text-input {
     right: 0;
   }
 }
+
+/* disable user select globally for whiteboard on iOS/iPad. Is there a better option? */
+html:is(.is-ios, is-native-ios, is-native-ipad) [data-page="whiteboard"] * {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

```suggestion
  user-select: none;
```
Is this going to be enough, assuming that postcss autoprefixer works properly?

---

Last time I checked it seems postcss did not transform them. I will do a double check.

---

The `autoprefixer` is not being configured at the root level. I will add that in another PR